data/tutorials/gs_00_up_and_running.md

```diff
@@ -27,90 +27,119 @@ for the instruction.
 
+The preferred way to install opam is with your system's package manager on
+Linux (e.g., `apt-get install opam` or similar). [Details of all installation
+methods.](https://opam.ocaml.org/doc/Install.html)
+
 ```shell
 # Ubuntu
-add-apt-repository ppa:avsm/ppa
-apt update
-apt install opam
+$ add-apt-repository ppa:avsm/ppa
+$ apt update
+$ apt install opam
 
 # Archlinux
-pacman -S opam
+$ pacman -S opam
 
 # Debian (stable, testing and unstable)
-apt-get install opam
+$ apt-get install opam
 ```
 
-**Binary distribution**
+**Binary Distribution**
 
 ```shell
-bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
+$ bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
 ```
 
-### Initialising `opam`
+### Initialising opam
+
+It's essential to both initialise opam and install the OCaml compiler because OCaml is a compiled language, so please don't skip this section. 
 
 This step is done automatically by the alpha-version platform installer, [see below](#up-and-running-with-the-platform-installer).
 
-If you have installed the binary distribution of `opam` through the install script, this step should already be done. If you have installed it through your system package manager, you must initialise `opam` by running the following command:
+If you have installed the binary distribution of opam through the install script, this step should already be done. If you have installed it through your system package manager, you must initialise opam by running the following command:
 
-```
-opam init          # Can take some time
-eval $(opam env)
+```shell
+$ opam init          # Can take some time
+$ eval $(opam env)
 ```
 
-The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. If you have installed OCaml through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of OCaml.
+The first command (`opam init`) creates a first switch, usually called `default`, although this is just a convention. (`switch` is used to have several installations on disk, like packages, compiler version, etc.) If you have installed OCaml through your system package manager, the first switch will be set up to use this compiler (it is called a "system switch"). Otherwise, it will build one from source, usually taking the most recent version of OCaml.
 
 The second command (`eval $(opam env)`) modifies a few environments variables to make the shell aware of the switch you are using. For instance, it will add what is needed to the `PATH` variable so that typing `ocaml` in the shell runs the OCaml binary of the current switch.
 
-In case you are not satisfied with the OCaml version of your system switch, you can write the following commands to create a new switch with a recent version of OCaml:
```

Comment:
Why did you remove this? I think it is worth it specifying that we not always want to create a new switch, only if we want a specific version of OCaml. (it is quite long to create a new switch)

Reply:
Good suggestion. Does changing the Install Compiler section to Creating a New Switch along with your other suggestion of adding "If you want a specific version of OCaml..." solve this problem?

Reply:
Yes, I think it solves the problem!

Reply:
Great! Thanks for the feedback!